### PR TITLE
fix: Check if variant is default to add suffix in docs property

### LINKED ISSUE #685: Documentation link for default plugins in CLI should drop variant
If a plugin is the default specified, the `To learn more about <plugin> <name>, visit <url>` should not have the `--variant` appended to it.

### PR DESCRIPTION
**Pull request by edgarrmondragon**
Closes #685

### CODE REVIEW

**Comment by edgarrmondragon** [comment]
```console
$ http https://deploy-preview-686--meltano-hub.netlify.app/meltano/api/v1/plugins/extractors/tap-postgres--transferwise | jq '.docs'
"https://deploy-preview-686--meltano-hub.netlify.app/extractors/tap-postgres"

$ http https://deploy-preview-686--meltano-hub.netlify.app/meltano/api/v1/plugins/extractors/tap-postgres--singer-io | jq '.docs'   
"https://deploy-preview-686--meltano-hub.netlify.app/extractors/tap-postgres--singer-io"
```

**Review by tayloramurphy — APPROVED** [review]

**Comment by tayloramurphy** [comment]
@pnadolny13 can you take a quick look at this then merge if it's good to your eyes?

File: _plugins/meltano_json_generator.rb [hunk 1]
```diff
@@ -58,6 +43,24 @@ def generate(site)
       top_level_index[plugin_type] = plugin_type_index
     end
     generate_file(site, "/meltano/api/v1/plugins", "index", JSON.generate(top_level_index))
+
+    # Write plugin files
```
Comment (edgarrmondragon) [inline _plugins/meltano_json_generator.rb:47]: I moved this after the index endpoint is populated so I can use the `defaults` mapping to get the default variant.

**Comment by pnadolny13** [comment]
@edgarrmondragon this looks good to me! I'll let you hit the merge button.

**Review by pnadolny13 — APPROVED** [review]

**Comment by edgarrmondragon** [comment]
@tayloramurphy note that for plugins already locked, the suffix will still be shown. A `meltano lock --update --all` would fix it.

**Pull request merged**